docs/docs/installation/kustomize.md

```diff
@@ -81,7 +81,33 @@ resources:
 
 ```
 
-#### ArgoCD
+## Configuration
+
+Kustomize allows for customization through overlays. For example: if you want to
+change log format to JSON, you can apply an override to the container and provide the
+required arguments:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - https://github.com/grafana/grafana-operator/releases/download/{{<param version>}}/kustomize-cluster_scoped.yaml
+
+patches:
+  - target:
+      group: apps
+      version: v1
+      kind: Deployment
+      name: grafana-operator-controller-manager
+    patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: -zap-encoder=json
```

Comment:
```suggestion
        value: --zap-encoder=json
```
Both `--` and `-` are fine in this case (tested), but let's use `--` to make it match operator-sdk [docs](https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/#default-zap-logger).